app/controllers/recommendations_controller.rb

```diff
@@ -56,9 +56,9 @@ def base_object
       Recommendation
     end
 
-    return records if params[:include_archive] != "false"
-
-    records.where(is_archive: false)
+    records = records.where(is_archive: false) if params[:include_archive] == "false"
+    records = records.where(id: records.select(&:is_current).map(&:id)) if params[:current_only] == "true"
+    records
```

Comment:
likewise, measures, indicators and progress reports should also be limited to current items if query param is set. no need to limit categories however (at least for now)